packages/web-components/src/components/va-accordion/va-accordion-item.css

```diff
@@ -61,8 +61,7 @@ button:hover {
   border-left: var(--item-border);
   border-right: var(--item-border);
   border-bottom: var(--item-border);
-  display: flex;
-  flex-wrap: wrap;
```

Comment:
This was not a good solution for the problem because it created defects when children content didn't have a width of 100%. This caused elements like `<p>` to be displayed inline instead of at block level like previously. Using this previous solution would have required us to make unnecessary updates to account for new children elements.